roles/setup_replication/tasks/configure_node.yml

```diff
@@ -10,6 +10,28 @@
         create: true
   no_log: "{{ disable_logging }}"
 
+- name: Store edb_master_key in a secure location
+  ansible.builtin.copy:
+    content: "{{ edb_master_key }}"
+    dest: "{{ edb_master_key_location_file }}"
```

Comment:
`{{ edb_master_key_plain_file }}` instead of variable `edb_master_key_location_file` which does not exist.